README.md

```diff
@@ -0,0 +1,80 @@
+# sorah-rbpkg/actions
+
+## Features
+
+- Install the package according to the platform from [Software Repositories - sorah.jp](https://sorah.jp/packaging/debian/).
+- Cache RubyGems installed with Bundler.
+
+## Usage
+
+See also [action.yml](action.yml)
+
+### Basic
+
+You can specify the Ruby version in the input `ruby-version`.
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: sorah-rbpkg/actions@v1
+    with:
+      ruby-version: "3.2"
+  - run: ruby app.rb
+```
+
+Only the following versions are supported. Specify it as a string.
+
+- `3.0`
+- `3.1`
+- `3.2`
+
+### `.ruby-version` file support
+
+You can install Ruby based on a file describing the Ruby version committed to a repository.
+Specifying `ruby-version-file` behaves the same as specifying `ruby-version` with the contents of the file at the specified path.
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: sorah-rbpkg/actions@v1
+    with:
+      ruby-version-file: .ruby-version
+  - run: ruby app.rb
+```
+
+If you want to specify `.ruby-version` in `ruby-version-file`, you can omit the input. Neither `ruby-version` nor `ruby-version-file` should be specified.
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: sorah-rbpkg/actions@v1
+  - run: ruby app.rb
+```
+
+## Bundler cache
+
+This action allows you to automatically install, cache, and restore dependencies specified in your Gemfile.
+To use this feature, you need to pre-generate Gemfile.lock and commit it to your repository. Then specify the input `bundler-cache` to true.
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: sorah-rbpkg/actions@v1
+    with:
+      bundler-cache: true
+  - run: bundle exec ruby app.rb
+```
+
+## Ruby package version
+
+You can explicitly specify the Ruby package version using the input `ruby-package-version`.
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: sorah-rbpkg/actions@v1
+    with:
+      ruby-version: '3.1'
+      ruby-package-version: 3.1.3-0nkmi1~jammy
```

Comment:
We can determine `ruby-version` from source version part of `ruby-package-version` and no one wants to specify debian version. Could you change the code to accept `X.Y.Z` format in `ruby-version`?

Reply:
Support patch version 43f6ceb cfe34d7 a124125